frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/OptionTabs.test.tsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import { ComponentType } from 'app-shared/types/ComponentType';
+import type { FormItem } from '../../../../../types/FormItem';
+import type { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+import userEvent from '@testing-library/user-event';
+import { renderWithProviders } from '../../../../../testing/mocks';
+import { OptionTabs } from './OptionTabs';
+import { componentMocks } from '@altinn/ux-editor/testing/componentMocks';
+import {
+  addFeatureFlagToLocalStorage,
+  removeFeatureFlagFromLocalStorage,
+} from 'app-shared/utils/featureToggleUtils';
+
+// Test data:
+const mockComponent = componentMocks[ComponentType.RadioButtons];
+
+describe('EditOptions', () => {
+  afterEach(() => jest.clearAllMocks());
+
+  it('should render component', async () => {
+    renderEditOptions();
+    expect(screen.getByText(textMock('ux_editor.options.tab_code_list'))).toBeInTheDocument();
+  });
+
+  it('should show code list input by default when neither options nor optionId are set', async () => {
```

Comment:
Nice test case! But async can be removed:
```suggestion
  it('should show code list input by default when neither options nor optionId are set', () => {
```